pkg/loader/translate.go

```diff
@@ -84,6 +86,17 @@ func columnToArg(mysqlType string, c *pb.Column) (arg interface{}) {
 			var str string = string(c.GetBytesValue())
 			return str
 		}
+		// issues: TOOL-1346 and TOOL-1374
+		// for downstream = mysql
+		// it work for tidb to update binary by translating []byte to uint64
```

Comment:
replace comments like this:
```
https://github.com/pingcap/tidb/issues/10988
Binary literal is not passed correctly for TiDB in some cases, so encode BIT types as integers instead of byte strings. Since the longest value is BIT(64), it is safe to always convert as uint64
```